pkg/http/controllers.go

```diff
@@ -5,7 +5,7 @@ import (
 	"fmt"
 	"net/http"
 
-	"github.com/dsauerbrun/cragcast/pkg/weather-client"
+	client "github.com/dsauerbrun/cragcast/pkg/weather-client"
```

Comment:
No need to explicitly set the identifier `client` here for this import declaration. With the import statement `"github.com/dsauerbrun/cragcast/pkg/weather-client"`, everything after the `-` will be the identifier. Meaning `client` will already be the identifier.

Reply:
ah, I think this was automatically done by OD's linting config that I have installed on vs code.